docs/index.html

```diff
@@ -1135,6 +1135,34 @@ <h2 id="inert-heading">works with inert elements</h2>
       </p>
     </div>
 
+
+    <div id="demo-dom-remove">
+      <h2 id="dom-remove-heading">Pulls focus when removing DOM elements</h2>
+      <p>
+      </p>
+      <p>
+        <button id="activate-dom-remove" aria-describedby="dom-remove-heading">
+          activate trap
+        </button>
+      </p>
+      <div id="dom-remove" class="trap">
+        <p>
+          Here is a focus trap <a href="#">with</a> <a href="#">some</a> <a href="#">focusable</a> parts.
+        </p>
+        <p>
+          This button will be removed when clicking it: <button id="dom-remove-button">remove</button>
+        </p>
+        <p>
+          <button id="deactivate-dom-remove" aria-describedby="dom-remove-heading">
+            deactivate trap
+          </button>
+        </p>
+      </div>
+      <p>
+        <a href="https://github.com/focus-trap/focus-trap/blob/master/docs/js/default.js">View demo source <span aria-hidden="true">&gt;&gt;</span></a>
```

Comment:
`default.js` should be `dom-remove.js` here

Reply:
:+1: Fixed in [c8756d7](https://github.com/focus-trap/focus-trap/pull/932/commits/c8756d74df588f37055740df7f0c536d1a2c90cc)